DSharpPlus.Commands/CommandsExtension.cs

```diff
@@ -207,6 +207,7 @@ public void AddCommands(IEnumerable<Type> types, params ulong[] guildIds)
         {
             if (type.GetCustomAttribute<CommandAttribute>() is not null)
             {
+                this.Client.Logger.LogInformation("Adding command from type {Type}", type.FullName ?? type.Name);

```

Comment:
do we *really* want to log at info for every single command? perhaps debug instead?